wp-includes/widgets/class-wp-widget-media-gallery.php

```diff
@@ -0,0 +1,231 @@
+<?php
+/**
+ * Widget API: WP_Widget_Media_Gallery class
+ *
+ * @package WordPress
+ * @subpackage Widgets
+ * @since 4.8.0
+ */
+
+/**
+ * Core class that implements a gallery widget.
+ *
+ * @since 4.8.0
+ *
+ * @see WP_Widget
+ */
+class WP_Widget_Media_Gallery extends WP_Widget_Media {
+
+	/**
+	 * Constructor.
+	 *
+	 * @since  4.8.0
+	 * @access public
+	 */
+	public function __construct() {
+		parent::__construct( 'media_gallery', __( 'Gallery' ), array(
+			'description' => __( 'Displays an image gallery.' ),
+			'mime_type'   => 'image',
+		) );
+
+		$this->l10n = array_merge( $this->l10n, array(
+			'no_media_selected' => __( 'No images selected' ),
+			'select_media' => _x( 'Select Images', 'label for button in the gallery widget; should not be longer than ~13 characters long' ),
+			'change_media' => _x( 'Add Image', 'label for button in the gallery widget; should not be longer than ~13 characters long' ),
+			'edit_media' => _x( 'Edit Gallery', 'label for button in the gallery widget; should not be longer than ~13 characters long' ),
+			'missing_attachment' => sprintf(
+				/* translators: placeholder is URL to media library */
+				__( 'We can&#8217;t find that gallery. Check your <a href="%s">media library</a> and make sure it wasn&#8217;t deleted.' ),
+				esc_url( admin_url( 'upload.php' ) )
+			),
+			'media_library_state_multi' => '',
+			'media_library_state_single' => '',
+		) );
+	}
+
+	/**
+	 * Get schema for properties of a widget instance (item).
+	 *
+	 * @since  4.8.0
+	 * @access public
+	 *
+	 * @see WP_REST_Controller::get_item_schema()
+	 * @see WP_REST_Controller::get_additional_fields()
+	 * @link https://core.trac.wordpress.org/ticket/35574
+	 * @return array Schema for properties.
+	 */
+	public function get_instance_schema() {
+		return array_merge(
+			parent::get_instance_schema(),
```

Comment:
`get_instance_schema()` here will return `title`, `attachment_id`, and `url`.  We do need the title, but not `attachment_id` nor `url`.  Should we even call the parent method here, or just simply add in the `title` bits directly?